keps/prod-readiness/sig-node/213.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 213
+stable:
+  approver: "@johnbelamaric"
```

Comment:
None of the PRR questions have been answered, please copy them into the KEP and answer them. Even the older ones about enablement, because it will be a long time before folks are all to 1.21 and having those docs can help people on older versions.

Reply:
added , PTAL @johnbelamaric